tools/ci_build/github/azure-pipelines/templates/win-gpu-ci.yml

```diff
@@ -109,6 +113,14 @@ jobs:
         workingDirectory: '$(Build.SourcesDirectory)\cmake\external\onnx'
         displayName: 'Install ONNX'
 
+      - ${{ if eq(parameters.isTraining, true) }}:
+          - script: |
+             python -m pip install -q wheel numpy
```

Comment:
is this installation done already?
https://github.com/microsoft/onnxruntime/blob/e096fbab92f12eb281742aa6b1654b30ab7d6b06/tools/ci_build/github/azure-pipelines/templates/win-gpu-ci.yml#L101